README.md

```diff
@@ -45,6 +45,28 @@ an improvement to the code, for example).
 If you're having trouble getting your output to match the expected, post an
 issue.
 
+## Using srclib-ruby in Windows/Cygwin environment
+
+### Prerequisites
+Windows with Cygwin installed, I have tested it with 
+```
+CYGWIN_NT-6.1-WOW Diana 2.0.4(0.287/5/3) 2015-06-09 12:20 i686 Cygwin
+```
+* Download and install Ruby 2.2.2 using [RubyInstaller](http://rubyinstaller.org/downloads/). I have tried [x64 version]((http://dl.bintray.com/oneclick/rubyinstaller/rubyinstaller-2.2.2.exe)) Make sure that ruby.exe is included into your path.
+* Download Development Kit from [RubyInstaller](http://rubyinstaller.org/downloads/). I have used [x64 version]( http://dl.bintray.com/oneclick/rubyinstaller/DevKit-mingw64-64-4.7.2-20130224-1432-sfx.exe). Extract self-extracting archive somewhere (say `C:\rbdevkit`), switch to this directory.
+* Run ```ruby dk.rb init``` to generate `config.yml`
+* Edit `config.yml` and add a line like ` - RUBY-INSTALLATION-DIR`, for example ` - C:/ruby22-x64`
+* Run ```ruby dk.rb install```
+* Run ```gem install bundler -v 1.6.9```, using 1.6.x is important because srclib-ruby is supposed to work with 1.6.x.
+* Run ```gem install gem-exefy``` - this tool replaces .bat files in RubyInstaller with .exe. Ensure that you have installed DevKit as described above, otherwise gem wouldn't work.
+* Run ```gem exefy --all``` to convert .bat files to .exe in RubyInstaller (for example, it will create `bundle.exe` and remove `bundle.bat`
+* Now, you are ready to run ```make``` in your srclib-ruby directory. It will take some time while Ruby downloads required files and then Yard builts cache of Ruby 2.2.2 files.
```

Comment:
^ Can the above steps be scripted so the user just has to run one command? Preferably, this will be run on `make`, too, so the installation steps are the same on Windows vs *nix.